### PR TITLE
Update secure-score-comparison-bar.png

### PR DESCRIPTION
**Pull request by msbemba**
Changed the Screenshot as it has changed recently.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6841

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/microsoft-secure-score-history-metrics-trends.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ In the **Metrics & trends** tab, there are several graphs and charts to give you
 
 ### Compare your score to organizations like yours
 
-There are two places to see how your score compares to organizations that are similar to you. In both charts, you can select **Manage comparisons** to view and edit your organization's information. You can also create a custom comparison based on industry, organization size, licenses, and regions.
+There are two places to see how your score compares to organizations that are similar to you.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/microsoft-secure-score-history-metrics-trends.md]:
```suggestion
There are two places to see how your score compares to organizations that are similar to yours.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**